lib/ldp.js

```diff
@@ -324,7 +324,7 @@ class LDP {
   async get (options, searchIndex = true) {
     let path, contentType, stats
     try {
-      ({ path, contentType } = await this.resourceMapper.mapUrlToFile({ url: options, searchIndex }))
+      ({ path, contentType } = await this.resourceMapper.mapUrlToFile({ url: options, contentType: options.contentType, searchIndex }))
```

Comment:
(See comment below first.)

Is `options.contentType` the value of the `Accept` header? Then it's not a content type, so can't be passed. But in any case, passing the content type to the resource mapper is only meaningful in the case of `PUT` or `POST`, when a new URL needs to be created; the mapper does not do content negotiation.

It seems to me that the desired logic should be implemented at this level, perhaps as follows:
- Set `searchIndex` to `false` if the `Accept` header does not explicitly ask for `text/html`.
- Then call `mapUrlToFile`.

Reply:
Ah okay thanks. I'll do that